routing/cross_mwm_connector.hpp

```diff
@@ -13,25 +14,153 @@
 
 namespace routing
 {
+// @todo(bykoianko) This using should be moved to a special file with TransitId class.
+// Identifier to find and a border edge in neighbouring mwm while cross mwm transition
+// for pedestrian, bicycle and car routing.
+using OsmId = uint64_t;
```

Comment:
Can we use Newtypes here?

Reply:
I'll check but I guess yes if we add a hash function for it to use it as a key of unordered_map. Do you propose to have a real type for routing::transit::OsmId to let the compiler to check more?